### PR TITLE
Refactor python MAGE module dev tutorial

### PR DESCRIPTION
**Pull request by antoniofilipovic**

### CODE REVIEW

File: mage/tutorials/create-a-new-module-python-local.mdx [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+> Note that query modules are loaded into Memgraph on startup so if your instance was already running you will need to
+> execute the following query inside one of [querying platforms](https://docs.memgraph.com/memgraph/connect-to-memgraph) to load them:
+```
+CALL mg.load_all();
+```
+
+Lastly, run a query and test your module:
+
+```
+MERGE (start:Node {id: 0})-[:RELATION]->(mid:Node {id: 1})-[:RELATION]->(end:Node {id: 2})
```
Comment (jmatak) [inline mage/tutorials/create-a-new-module-python-local.mdx:227]: Why we have mid and end if we are not using it :sweat_smile:

**Review by jmatak — CHANGES_REQUESTED** [review]
Overall great. Fix the two issues I have commented

File: mage/tutorials/create-a-new-module-python-local.mdx (hunk 1 above)
Comment (g-despot) [reply mage/tutorials/create-a-new-module-python-local.mdx]: Removed `mid` and `end`.

File: mage/tutorials/create-a-new-module-python-local.mdx [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+
+```
+exit
+```
+:::
+
+  </TabItem>
+
+  <TabItem value="source">
+**0.** Make sure your Memgraph instance is running.
```
[the remark this answers is not in the document]
Comment (g-despot) [reply mage/tutorials/create-a-new-module-python-local.mdx]: Fixed formatting.

**Review by g-despot — APPROVED** [review]

**Pull request merged**